google-cloud-language/lib/google/cloud/language/document.rb

```diff
@@ -70,10 +70,16 @@ def content?
         ##
         # @private Whether the document source is a Google Cloud Storage URI.
         #
-        def url?
+        def gcs_url?
           @grpc.source == :gcs_content_uri
         end
 
+        def url?
```

Comment:
Ah Yes. I was being a bit overly protective, trying to not break users doing some bad OOP things (calling private instance methods through #send). I removed it.